tests/test_core.py

```diff
@@ -156,9 +156,8 @@ def test_mon_climatology(ds_name, decoded, variables, time_coord_name):
     ds = esmlab.datasets.open_dataset(ds_name, decode_times=decoded)
     esm = ds.esmlab.set_time(time_coord_name=time_coord_name)
```

Comment:
is `esm` an xarray object, if so use `xarray.testing._assert_internal_invariants(esm)` to make sure you aren't breaking any assumptions

Reply:
`esm` here is an instance of `esmlab.core.EsmlabAccessor`

Reply:
so check `esm._ds`?

Reply:
> so check esm._ds?

Done.